packages/components/react/src/Panels/PreviewPanel.tsx

```diff
@@ -160,6 +160,10 @@ function Preview({ preview, iframe, previewCount, first, last, toggleTerminal, i
     if (preview.url) {
       iframe.ref.src = preview.url;
     }
+
+    if (preview.title) {
+      iframe.ref.title = preview.title;
```

Comment:
`title` in iframes is special case:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#accessibility
- https://dequeuniversity.com/tips/provide-iframe-titles